video_creation/final_video.py

```diff
@@ -103,12 +103,34 @@ def prepare_background(reddit_id: str, W: int, H: int) -> str:
         exit()
     return output_path
 
+def merge_background_audio(audio: ffmpeg, reddit_id: str):
+    """Gather an audio and merge with assets/backgrounds/background.mp3
+    Args:
+        audio (ffmpeg): The TTS final audio but without background.
+        reddit_id (str): The ID of subreddit
+    """
+    background_audio_volume = settings.config["settings"]["background"]["background_audio_volume"]
+    if (background_audio_volume == 0):
+        return audio # Return the original audio
+    else:
+        # sets volume to config
+        bg_audio = (
+            ffmpeg.input(f"assets/temp/{reddit_id}/background.mp3")
+            .filter(
+                "volume",
+                background_audio_volume,
+            )
+        )
+        # Merges audio and background_audio
+        merged_audio = ffmpeg.filter([audio, bg_audio], "amix", duration="longest")
```

Comment:
hmmm odd but sure.

Reply:
When i was merging with `develop branch`  i kept the `merged_audio` function from `develop branch`.
Just take a look:
It's from develop branch
![image](https://user-images.githubusercontent.com/105776775/234146274-9bd7a556-4bc5-4f1c-9aba-e899525dd91a.png)